src/main/java/frc/robot/commands/Shoot.java

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.subsystems.Shooter;
+
+public class Shoot extends CommandBase {
+  /** Creates a new Shoot. */
+  Shooter shooter;
+  public Shoot(Shooter shooter) {
+    // Use addRequirements() here to declare subsystem dependencies.
+    addRequirements(shooter);
+    this.shooter = shooter;
+  }
+
+  // Called when the command is initially scheduled.
+  @Override
+  public void initialize() {}
```

Comment:
```suggestion
  public void initialize() {
    shooter.shoot(-1);
  }
```
Once we set the speed of the motor, it'll be at that speed until told otherwise, so setting the motor's speed only once is sufficient in this case. Initialize() would then be preferable since it'll only run once, and execute() actually runs every 20 ms.